.github/workflows/swift.yml

```diff
@@ -0,0 +1,15 @@
+name: ImageRow Swift Package
+
+on: [push]
```

Comment:
Why not running the action on `pull_request`?

Reply:
You are right, the GA should also run on pull requests. I added this change to the PR.